comms/discovery/rpcz/chat.go

```diff
@@ -239,14 +267,22 @@ func chatSetPermissions(tx *sqlx.Tx, userId int32, permits schema.ChatPermission
 		return err
 	}
 
+	if isInPermitList(schema.All, permitList) {
+		err := updatePermissions(tx, userId, schema.All, true, messageTimestamp)
+		return err
+	} else if isInPermitList(schema.None, permitList) {
+		return nil
```

Comment:
`none` is not really an option in the UI right... so we make this a noop?

Reply:
yeah was thinking about that.. maybe we should have client explicitly send none when no options selected

Reply:
if we do that, seems like we'd want to add a row saying "none" here as well?